### PR TITLE
Added option 'theme'

### PR DESCRIPTION
**Pull request by lmeysel**
Required the added feature for one of my projects - not the best solution, but it does for my purposes. Maybe you're interested.

### CODE REVIEW

**Comment by tangshuang** [comment]
hi @lmeysel could you make your editor tab space as 2 space, now I do not know which lines have you modified. thx!

**Comment by lmeysel** [comment]
sorry for that - fixed it

File: src/gulp-css-copy-assets.js [hunk 1]
```diff
@@ -41,7 +41,12 @@ export default function(options = {}) {
                   return
                 }
                 // clear ' or  '
-                let fileurl = url.replace('"', '').replace("'", '')
+                var fileurl = url.replace(/\"/g, '').replace(/\'/g, '');
```
Comment (tangshuang) [inline src/gulp-css-copy-assets.js:44]: This is a nice bugfix

File: src/gulp-css-copy-assets.js [hunk 2]
```diff
@@ -41,7 +41,12 @@ export default function(options = {}) {
                   return
                 }
                 // clear ' or  '
-                let fileurl = url.replace('"', '').replace("'", '')
+                var fileurl = url.replace(/\"/g, '').replace(/\'/g, '');
+                var qry = fileurl.indexOf('?'), qrystring = '';
+                if(qry !== -1) {
+                   qrystring = fileurl.substring(qry);
+                  fileurl = fileurl.substring(0, qry);
+                }
```
Comment (tangshuang) [inline src/gulp-css-copy-assets.js:49]: And query string is missed by me.

**Pull request merged**